packages/subgraph/src/commons.ts

```diff
@@ -60,6 +60,18 @@ export function templateId(onChainId: BigInt, onChainVersion: BigInt): Bytes {
     );
 }
 
+export function featureId(
+    managerAddress: Bytes,
+    templateId: BigInt,
+    featureId: BigInt,
+): Bytes {
+    return managerAddress.concat(
```

Comment:
Can we add a sort of divider when generating the `featureId`? Something like `manager_address-template_id-feature_id`.